relnotes.txt

```diff
@@ -133,27 +147,28 @@ Of Fame" [13].
 
 ACKNOWLEDGEMENTS
 
-This is the seventeenth release of Tahoe-LAFS to be created
+This is the eighteenth release of Tahoe-LAFS to be created
 solely as a labor of love by volunteers. Thank you very much
 to the team of "hackers in the public interest" who make
 Tahoe-LAFS possible.
 
 Brian Warner
 on behalf of the Tahoe-LAFS team
 
-January 18, 2017
+May 17, 2018
 San Francisco, California, USA
 
 
-[1] https://github.com/tahoe-lafs/tahoe-lafs/blob/tahoe-lafs-1.12.1/NEWS.rst
+[1] https://github.com/tahoe-lafs/tahoe-lafs/blob/tahoe-lafs-1.13.0/NEWS.rst
 [2] https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/known_issues.rst
 [3] https://tahoe-lafs.org/trac/tahoe-lafs/wiki/RelatedProjects
-[4] https://github.com/tahoe-lafs/tahoe-lafs/blob/tahoe-lafs-1.12.1/COPYING.GPL
-[5] https://github.com/tahoe-lafs/tahoe-lafs/blob/tahoe-lafs-1.12.1/COPYING.TGPPL.rst
-[6] http://tahoe-lafs.readthedocs.org/en/tahoe-lafs-1.12.1/INSTALL.html
+[4] https://github.com/tahoe-lafs/tahoe-lafs/blob/tahoe-lafs-1.13.0/COPYING.GPL
+[5] https://github.com/tahoe-lafs/tahoe-lafs/blob/tahoe-lafs-1.13.0/COPYING.TGPPL.rst
+[6] http://tahoe-lafs.readthedocs.org/en/tahoe-lafs-1.13.0/INSTALL.html
 [7] https://tahoe-lafs.org/cgi-bin/mailman/listinfo/tahoe-dev
 [8] https://tahoe-lafs.org/trac/tahoe-lafs/roadmap
 [9] https://github.com/tahoe-lafs/tahoe-lafs/blob/master/CREDITS
 [10] https://tahoe-lafs.org/trac/tahoe-lafs/wiki/Dev
 [12] https://leastauthority.com/
 [13] https://tahoe-lafs.org/hacktahoelafs/
+[14] http://magic-wormhole.io
```

Comment:
Hmm.  No https here though. :(

But it's only a redirect to github.  Should we just link to github directly?  https://github.com/warner/magic-wormhole